openlibrary/core/imports.py

```diff
@@ -116,6 +116,16 @@ def find_by_identifier(identifier):
         if result:
             return ImportItem(result[0])
 
+    @staticmethod
+    def bulk_mark_pending(identifiers: list[str]):
+        query = (
+            "UPDATE import_item"
+            "SET status = 'pending'"
+            "WHERE status = 'staged'"
```

Comment:
```suggestion
            "UPDATE import_item "
            "SET status = 'pending' "
            "WHERE status = 'staged' "
```
This may or may not need some spaces so that it doesn't become `"UPDATE import_itemSET status...`, etc.

Reply:
I don't think that the newline characters are stripped out, so this should work as expected.  Will confirm when I test this later today.